docs/guidance/rayservice-troubleshooting.md

```diff
@@ -0,0 +1,81 @@
+# RayService troubleshooting
+
+RayService is a Custom Resource Definition (CRD) designed for Ray Serve. In KubeRay, it first creates a RayCluster and then proceeds to 
```

Comment:
Should we say instead that "RayService is a Custom Resource"? It feels a bit weird to say it is a CRD, I think of it like there is a CRD for RayService, by RayService itself is a CR. LMK if this is accurate. Also, would it be helpful to link to the kubernetes Custom Resource docs?

Reply:
The relationship between CRD and CR is similar to class and instance.